tests/JustSaying.IntegrationTests/Fluent/Subscribing/WhenTwoDifferentHandlersHandleAMessage.cs

```diff
@@ -19,8 +18,11 @@ public WhenTwoDifferentHandlersHandleAnExactlyOnceMessage(ITestOutputHelper outp
         public async Task Then_Both_Handlers_Receive_The_Message()
         {
             // Arrange
-            var handler1 = new ExactlyOnceHandlerNoTimeout();
-            var handler2 = new ExactlyOnceHandlerNoTimeout();
+            var nullStoreLogger = NullLogger<TestMessageStore<SimpleMessage>>.Instance;
+            var nullHandlerLogger = NullLogger<MessageStoringHandler<SimpleMessage>>.Instance;
+
+            var handler1 = new MessageStoringHandler<SimpleMessage>(new TestMessageStore<SimpleMessage>(nullStoreLogger), nullHandlerLogger);
```

Comment:
If you wanted to simplify this a little, there's also an `InspectableHandler<T>` that just collects messages on the handler, instead of using a separate store. MessageStoringHandler is useful when using DI to resolve a handler instead of creating it like here :)

Reply:
Thanks, yeah I'll take a look, this felt a bit clunky